src/components/footer.tsx

```diff
@@ -17,20 +17,25 @@ import { HelpCenterLink } from './help-center-link';
 import { UnsubscribeLink } from './unsubscribe-link';
 
 const socialElements = [
+  {
+    key: 'instagram',
+    href: 'https://www.instagram.com/3ookcom',
+    assetPath: '/social-elements/v2/instagram.png',
+  },
   {
     key: 'facebook',
-    href: 'https://www.facebook.com/Liker.Land',
+    href: 'https://www.facebook.com/3ookcom',
     assetPath: '/social-elements/v2/facebook.png',
   },
   {
-    key: 'instagram',
-    href: 'https://www.instagram.com/liker.land',
-    assetPath: '/social-elements/v2/instagram.png',
+    key: 'threads',
+    href: 'https://www.threads.com/@3ookcom',
+    assetPath: '/social-elements/v2/threads.png',
   },
   {
-    key: 'blog',
-    href: 'https://blog.liker.land',
-    assetPath: '/social-elements/v2/blog.png',
+    key: 'substack',
+    href: 'https://3ookcom.substack.com/',
```

Comment:
Should we just use `https://review.3ook.com/`

Reply:
Updated and merged